modules/AlphaWalletWeb3/AlphaWalletWeb3/Utils/Web3+Utils.swift

```diff
@@ -204,6 +204,13 @@ extension Web3.Utils {
         return Web3.Utils.personalECRecover(data, signature: sig)
     }
 
+    static public func personalECRecover(_ message: Data, r: [UInt8], s: [UInt8], v: UInt8) -> Data? {
```

Comment:
Better to call it `recoverPublicKey` instead of `personalECRecover` to reflect what it does

Reply:
Don't forget to rename this one too